01-node-tutorial/answers/.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules  
+.DS_Store
+/temp.txt
```

Comment:
Still did not target the right files
`./content/temp.txt` that where the file is located